modelon/impact/client/client.py

```diff
@@ -741,6 +741,22 @@ def get_me(self) -> User:
     def create_entity_from_reference(
```

Comment:
I suggest for user facing flow two functions marked "experimental":
```
def get_experiment_by_reference(...) -> Experiment
def get_case_by_reference(...) -> Case
```

Reply:
Makes sense! Done!